pkg/apis/compliance/v1alpha1/tailoredprofile_types.go

```diff
@@ -15,6 +15,9 @@ const PruneOutdatedReferencesAnnotationKey = "compliance.openshift.io/prune-outd
 // RuleLastCheckTypeChangedAnnotationKey is the annotation key used to indicate that the rule check type has changed, store its previous check type
 const RuleLastCheckTypeChangedAnnotationKey = "compliance.openshift.io/rule-last-check-type"
 
+// ExtendedProfileUniqueIDLable is a label used to store the unique ID of the profile being extends
+const ExtendedProfileUniqueIDLable = "compliance.openshift.io/extended-profile-unique-id"
```

Comment:
Do we need a new label for the extend profile? 
It seems to be that we could use the same label name `ProfileUniqueIDLabel`?
This will also help make it easier for label consumers, they won't need to handle profile and extended profiles differently when it comes to consuming their UniqueIDs.


Also typo on label:

```suggestion
const ExtendedProfileUniqueIDLabel = "compliance.openshift.io/extended-profile-unique-id"
```

Reply:
ExtendedProfileUniqueIDLabel label is only used for refer to profile being extended in the tailoredprofile, a `ProfileUniqueIDAnnotation` is also used in the tailordprofile

Reply:
Aha, I see. To be able to backtrack using the profile's unique ID.